lib/api/entities.rb

```diff
@@ -147,6 +147,11 @@ class ProjectEntity < Grape::Entity
       expose :state, :created_at, :updated_at
     end
 
+    class RepoDiff < Grape::Entity
```

Comment:
Note about this one: I had to move it otherwise the the `MergeRequestFiles` did not see this one.